### PR TITLE
Tracer service tests fix

### PR DESCRIPTION
**Pull request by romanova-natasha**

### CODE REVIEW

File: integration/tests/tracer/test_tracer_debug_methods.py [hunk 1]
```diff
@@ -505,10 +506,13 @@ def test_debug_get_modified_accounts_by_invalid_hash(self, params):
 
     def test_debug_get_raw_transaction(self):
         sender_account = self.accounts[0]
-        nonce = self.web3_client.get_nonce(sender_account.address)
-        transaction = self.web3_client.make_raw_tx(sender_account, nonce=nonce, amount=0.1)
+        transaction = self.web3_client.make_raw_tx(sender_account)
+        transaction["data"] = GOOD_CALLDATA[0]
+        transaction["chainId"] = self.web3_client.eth.chain_id
+        transaction["gas"] = self.web3_client.eth.estimate_gas(transaction)
```
Comment (kristinaNikolaevaa) [inline integration/tests/tracer/test_tracer_debug_methods.py]: chainId and gas were added by make_raw_tx(
and there are data parametr too 
transaction = self.web3_client.make_raw_tx(sender_account, data=GOOD_CALLDATA[0])

File: integration/tests/conftest.py [hunk 2]
```diff
@@ -394,6 +394,18 @@ def storage_contract(web3_client, class_account) -> tp.Any:
     yield contract
 
 
+@pytest.fixture(scope="class")
+def storage_contract_with_deploy_tx(web3_client, class_account) -> tp.Any:
```
Comment (kristinaNikolaevaa) [inline integration/tests/conftest.py:398]: 1) we have storage_contract fixture  with the same contract in this file
2) why do you need to return contract_deploy_tx? if you need to get contract address just use contract.address

Comment (romanova-natasha) [reply integration/tests/conftest.py:398]: I need contract deploy block for two of my tests, I can get it only from deploy tx. storage_contract fixture is used in many tests and it is not convenient to use it as a tuple in all these tests

File: integration/tests/tracer/test_tracer_debug_methods.py (hunk 1 above)
Comment (romanova-natasha) [reply integration/tests/tracer/test_tracer_debug_methods.py]: agree and fixed

**Review by kristinaNikolaevaa — APPROVED** [review]

**Pull request merged**